### PR TITLE
L76 soft standby

### PR DESCRIPTION
**Pull request by thebentern**
Will test it tomorrow

### CODE REVIEW

**Review by caveman99 — APPROVED** [review]

File: src/gps/GPS.cpp [hunk 1]
```diff
@@ -502,11 +503,22 @@ void GPS::setGPSPower(bool on, bool standbyOnly, uint32_t sleepTime)
             }
             msglen = gps->makeUBXPacket(0x02, 0x41, 0x08, gps->_message_PMREQ);
             gps->_serial_gps->write(gps->UBXscratch, msglen);
+        } else if (gnssModel == GNSS_MODEL_MTK) {
+            // Enter soft standby mode
+            LOG_DEBUG("L76K entering soft standby mode\n");
+            gps->_serial_gps->write("$PMTK161,0*28\r\n");
```
Comment (thebentern) [inline src/gps/GPS.cpp:509]: @neilhao this command seems correct based on the MTK datasheets and I see that SoftRF does the same thing. However, I'm not seeing any power savings with my ATGM336H module. Do you happen to know if they implement the standby command? I can't find any docs specific to the module.

Comment (code8buster) [reply src/gps/GPS.cpp:509]: Give "$PCAS12,65535*1E\r\n" a try on your personal branch @thebentern @neilhao if this command doesn't work for you. It's listed in the CASIC protocol manual as a standby control msg. Should make the receiver standby for 65535 seconds, if the module supports it.

File: src/gps/GPS.cpp [hunk 2]
```diff
@@ -502,11 +503,22 @@ void GPS::setGPSPower(bool on, bool standbyOnly, uint32_t sleepTime)
...
     } else {
         if (gnssModel == GNSS_MODEL_UBLOX) {
             gps->_serial_gps->write(0xFF);
             clearBuffer(); // This often returns old data, so drop it
+        } else if (gnssModel == GNSS_MODEL_MTK) {
+            LOG_DEBUG("L76K exit soft standby mode\n");
+            // Start vehicle mode (to exit standby mode)
+            gps->_serial_gps->write("$PCAS11,3*1E\r\n");
+            delay(250);
         }
```
Comment (caveman99) [inline src/gps/GPS.cpp:522]: if the flush() call resolves the problem it's needed here too

Comment (thebentern) [reply src/gps/GPS.cpp:522]: I added that for good measure when I was troubleshooting why the power consumption wasn't lowering. I can probably yank it honestly.

**Pull request closed without merging**